thesis-de/content-de.tex

```diff
@@ -16,18 +16,91 @@ \section{Erklärung}
 \newpage
 
 \section{Einführung}
+\label{sec:einfuehrung}
 Das Heinrich-Hertz-Europakolleg Bonn verlangt im fünften und sechsten Semester
 der Weiterbildung zum staatlich geprüften Informatiker eine fachbezogene
 Projektarbeit. Dieses Projekt wird in Gruppen von zwei bis vier Personen
-durchgeführt und soll fachliche Inhalte sowie Inhalte aus dem Projektmanagement
+durchgeführt und soll fachliche Inhalte sowie Techniken aus dem Projektmanagement
 kombinieren. Es handelt sich um eine praktische Arbeit. Jeder Abschnitt enthält
 ein Kürzel des Autors, hierbei bedeutet:
 \begin{outline}
   \1 {[MR]} erstellt von Marcel Reuter
   \1 {[NL]} erstellt von Nikolai Luis
   \1 {[TM]} erstellt von Tim Meusel
 \end{outline}
-\tm%
+
+Die Zusammensetzung dieses Teams entstand während der rugulären
+Unterrichtszeit im dritten Semester. Herr Meusel und Herr Luis
+sprachen während einem Abendessen über die Inhalte ihres täglichen
+Arbeitstages. Herr Meusel arbeitete zu diesem Zeitpunkt an einem Projekt
+im Bereich Cloud Hosting und Herr Luis war von Datenanalysen und großen
+Datenmengen auf Datenbanksystemen aufgeschlossen. Sehr schnell entstand
+aus diesen beiden Gesprächsinhalten die Softwareidee, welche in diesem
+Dokument vorgestellt wird. Da Herrn Meusel und Herrn Luis der Aufwand
+dieser Lösung bewusst war und beide keine Expertise im Bereich Frontend
+Entwicklung, bzw. Datenvisualisierung besaßen, stellten Sie die Idee ihren
+Mitstudenten vor. Herr Reuter zeigte dabei hohes Interesse und war vorallem
+vom englischsprachigen Teil des Projektes überzeugt. Die von den Dozenten
+definierten Vorgaben waren somit erfüllt und das Projekt konnte für das
+fünfte und sechste Semester angemeldet werden.
+
+Eine der größten Hürden in diesem Projekt war die fehlende Zeit für
```

Comment:
Das ist ein rückblick, das gehört nicht in die einleitung, höchstens in das fazit